collector/Dockerfile

```diff
@@ -39,13 +39,18 @@
 # You must configure the Cluster-Insight data collector as explained in the
 # accompanying ../README.md file before running it.
 
-# To run a container from this image in dev/test mode
+# To run a container from this image in dev/test mode as the master:
 # (Ctrl-C will stop and remove the container):
 #   sudo docker run --rm --net=host -p 5555:5555 --name cluster-insight kubernetes/cluster-insight python ./collector.py --debug
 #
 # To run a container from this image in detached production mode:
```

Comment:
To run a container from this image in detached production mode: -->
To run a container from this image in detached production mode as the master: